src/planscape/planscape/filters.py

```diff
@@ -0,0 +1,5 @@
+from django_filters import filters
+
+
+class CharArrayFilter(filters.BaseCSVFilter, filters.CharFilter):
+    pass
```

Comment:
Just double checking that this is just meant to pull in BaseCSVFilter & CharFilter and not be otherwise implemented?